pages/deployment/[uuid].tsx

```diff
@@ -28,6 +27,8 @@ import Link from 'next/link'
 import { useRouter } from 'next/router'
 import React, { MouseEvent, useEffect, useMemo, useState } from 'react'
 import { Elements } from 'react-flow-renderer'
+import { ModelDoc } from 'server/models/Model'
```

Comment:
should be "import { ModelDoc } from '../../server/models/Model'"

Reply:
fixed